NetKAN/LaunchPadLamp.netkan

```diff
@@ -0,0 +1,10 @@
+{
+    "spec_version" : "v1.0",
+    "identifier"   : "LaunchPadLamp",
+    "$kref"        : "#/ckan/github/droric/Launch-Pad-Lamp",
+    "license"      : "CC-BY-NC-SA-3.0",
+    "ksp_version"  : "1.3.0",
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/164677-13-launch-pad-lamp/"
+    }
```

Comment:
You'll need to add an install clause:

    "install": [
        {
            "find":"Launch Pad Lamp",
            "install_to": "GameData"
        }
    ]

Reply:
I gather you've re-done the zipfile so it has a matching folder to the mod identifier. That works fine.